### PR TITLE
SE-18775: recommended to turn off the persistent queue in batch jobs

### PR DESCRIPTION
**Pull request by nbedarkar**

### CODE REVIEW

File: modules/ROOT/pages/cloudhub-fabric.adoc [hunk 1]
```diff
@@ -133,6 +133,8 @@ Number of Workers: 1
 
 When you deploy an application containing batch jobs to CloudHub with persistent queues enabled, the batch jobs use CloudHub's persistent queuing feature for the batch queuing functionality.
```
Comment (hannanelson) [inline modules/ROOT/pages/cloudhub-fabric.adoc:134]:
```suggestion
When you deploy an application containing batch jobs to CloudHub with persistent queues enabled, the batch jobs use CloudHub persistent queuing for the batch queuing functionality.
```

**Review by nbedarkar — COMMENTED** [review]
LGTM.

File: modules/ROOT/pages/cloudhub-fabric.adoc [hunk 2]
```diff
@@ -129,15 +129,31 @@ Number of Workers: 1
...
 
 
-== Persistent Queuing Behavior for Applications Containing Batch Jobs
+== Persistent Queues for Applications Containing Batch Jobs
 
-When you deploy an application containing batch jobs to CloudHub with persistent queues enabled, the batch jobs use CloudHub's persistent queuing feature for the batch queuing functionality.
+When you deploy an application containing batch jobs to CloudHub with persistent queues enabled, the batch jobs use CloudHub persistent queuing for the batch queuing functionality.
 
-When using persistent queues, keep in mind the following limitations:
+If you enable persistent queues, keep in mind the following limitations:
```
Comment (sarathecoubian) [inline modules/ROOT/pages/cloudhub-fabric.adoc]: Might replace "keep in mind" with a non-idiom for translation purposes - maybe "note the following limitations"?

**Review by sarathecoubian — DISMISSED** [review]
Approved with an optional comment.

**Review by sarathecoubian — APPROVED** [review]

**Pull request merged**